src/main/java/com/amplitude/AmplitudeLog.java

```diff
@@ -0,0 +1,38 @@
+package com.amplitude;
+
+public class AmplitudeLog {
+
+    private static LogMode LOG_MODE = LogMode.DEBUG;
+
+    public static void setLogMode(LogMode logMode) {
```

Comment:
We should give user to set this when they create Amplitude class, which means we need to have an API for user in `Amplitude` to set the log level.

I don't see this is being called anywhere btw.